napari/layers/_data_protocols.py

```diff
@@ -81,6 +81,14 @@ def __getitem__(
     ) -> LayerDataProtocol:
         """Returns self[key]."""
 
+    @property
+    def size(self) -> int:
+        pass
```

Comment:
```suggestion
        raise NotImplementedError()
```

pass is not excluded from coverage

Reply:
That is inconsistant with the dtype and getitem, shoudl we:
 - [ ] be consistant and raise notimpl in all of them or,
 - [ ] just leave the body empty with a docstring ?

Reply:
I like "just leave ..." option